src/components/common/form/InputRounded.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { StyleSheet, TextInput, View } from 'react-native'
+import normalize from 'react-native-elements/src/helpers/normalizeText'
+import Icon from 'react-native-elements/src/icons/Icon'
+import { HelperText } from 'react-native-paper'
+
+/**
+ * TopBar - used To display contextual information in a small container
+ * @param {object} props - an object with props
+ * @param {boolean} props.hideBalance - if falsy balance will be displayed
+ * @param {function} props.push - pushes a route to the nav stack. When called, apps navigates to the specified ruote
+ * @param {React.Node} props.children
+ * @returns {React.Node}
+ */
+const InputRounded = ({ icon, iconColor, error, ...inputProps }) => (
+  <View style={styles.inputContainer}>
+    <View style={styles.iconInputContainer}>
+      <TextInput style={styles.input} {...inputProps} />
+      <View style={styles.suffixIcon}>
```

Comment:
This icon should be always aligned to the right. Perhaps using `display: 'flex'` and `justifyContent:'space-between'` in the container